script/sync_prod_to_gcp/Makefile

```diff
@@ -29,5 +33,15 @@ test/arxiv-development_arxiv-stanza.json:
 test/arxiv-development_sync-test.json:
 	op read "op://hs3xn7ldhg3pgrql5j524rgpee/w2wtsf5v7kahbngr64m43mciou/qrdfsd5gbnatjpv7zof6vwca4q" > $@
 
-${APP_CRED}:
-	gcloud auth application-default login
+sync-to-gcp-queue: CURRENT_PROJECT := $(shell gcloud config get project)
+
+sync-to-gcp-queue:
+	-gcloud pubsub topics create ${SYNC_TO_GCP_TOPIC} --message-retention-duration=31d --message-encoding=json --project ${ARXIV_PROJECT_ID} --schema p2p
+
+sync-to-gcp-sub: 
+	gcloud pubsub subscriptions create ${SYNC_TO_GCP_SUB_ID} --topic=${SYNC_TO_GCP_TOPIC} --ack-deadline=10 --min-retry-delay=60 --max-retry-delay=600 --max-delivery-attempts=100
+
+install-sync-to-gcp-service:
+	sudo -H install -m 0644 resource/systemd/sync-to-gcp@.service /etc/systemd/system/
+	sudo systemctl daemon-reload
+	@echo "Please run sudo systemctl enable sync-to-gcp@sync-node.service"
```

Comment:
Should we add `--now` to the `systemctl` call to ensure it is also started, not only enabled, otherwise one needs a reboot to activate it?

Reply:
You are right. However, this is one time only on one host CIT sync-node, and it is more or less a reminder to myself when I deploy this.
When it is started, the service starts consuming the queue, and did not want to start without monitoring the logging. 
I know I can start it with systemctl start. But, I'm pretty sure I want to start it as e-prints and by hand at first.

Reply:
Ok, thanks!